ods_ci/tests/Resources/Files/accelerator/qe_accelerator_profile_6.yaml

```diff
@@ -0,0 +1,17 @@
+# File to provision an Accelerator Profile instance in test:
+#   "Disable an Accelerator Profile From The Grid of the Accelerator Profile View"
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  generation: 1
+  name: qeacceleratorprofile6
+  namespace: redhat-ods-applications
+spec:
+  displayName: qe_accelerator_profile_6
+  enabled: true
```

Comment:
Shouldn't it be "false" (disable) ?

Reply:
As in the test I'm disabling the AP, I need to provision one enabled.